sdk/agrifood/azure-agrifood-farming/samples/async/sample_attachments_async.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""
+FILE: sample_attachments_async.py
+
+DESCRIPTION:
+    This sample demonstrates FarmBeats' capabaility of storing arbitrary files 
+    in context to the various farm hierarchy objects. 
+    We first attach some files onto a farmer and a farm, and then download all 
+    existing attachments for the farmer onto a local directory.
+
+USAGE:
+    ```python sample_attachments_async.py```
+
+    Set the environment variables with your own values before running the sample:
+    - `AZURE_TENANT_ID`
```

Comment:
nit: i'd give a little blurb about what each of these env vars should be, TA example [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_authentication.py#L26)